src/Resources/config/services.yml

```diff
@@ -1,4 +1,16 @@
 services:
+    contao.controller.abstract_csv_import:
+        class: Contao\CoreBundle\Controller\AbstractCsvImportController
+        abstract: true
+        calls:
```

Comment:
I have changed that but left the setters to allow service overriding by child classes.